lib/apiservers/engine/backends/system.go

```diff
@@ -211,6 +219,16 @@ func (s *System) SystemInfo() (*types.Info, error) {
 			customInfo := [2]string{systemOSVersion, vchInfo.HostOSVersion}
 			info.SystemStatus = append(info.SystemStatus, customInfo)
 		}
+		if len(vchConfig.InsecureRegistries) > 0 {
+			customInfo := [2]string{insecureRegistriesLabel, InsecureRegistries()}
+			info.SystemStatus = append(info.SystemStatus, customInfo)
+		}
+		if len(vchConfig.RegistryWhitelist) > 0 {
+			customInfo := [2]string{vchWhitelistMode, "enabled"}
```

Comment:
Should we have "disabled" as a value? Easier for parsing/doc'ing than saying that the line doesn't exist in one case.